docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -324,33 +324,65 @@ Release
         git checkout -b release/flocker-${VERSION} origin/master
         git push origin --set-upstream release/flocker-${VERSION}
 
-   - Create a ``flocker-${VERSION}.rb`` file by copying the last recipe file and renaming it for this release.
+   - Create a ``flocker-${VERSION}.rb`` recipe file.
 
-   - Update recipe file:
+     XXX This should be automated: https://clusterhq.atlassian.net/browse/FLOC-1150
 
-     - Update the version number:
+     The starting contents of the recipe should be similar to the following:
 
-       The version number is included in the class name with all dots and dashes removed.
-       e.g. ``class Flocker012 < Formula`` for Flocker-0.1.2
+     .. code-block:: ruby
 
-     - Update the URL:
+       require "formula"
 
-       The version number is also included in the ``url`` part of the recipe.
+       class Flocker030dev1 < Formula
+         homepage "https://clusterhq.com"
+         url "http://storage.googleapis.com/archive.clusterhq.com/downloads/flocker/Flocker-0.3.0dev1.tar.gz"
+         sha1 "227dc898121b46670631c4eeaeb7424f218925c1"
+         depends_on :python if MacOS.version <= :snow_leopard
 
-     - Update the ``sha1`` checksum. Retrieve it with ``sha1sum``:
+     The version number is included in the class name with all dots and dashes removed,
+     e.g. ``class Flocker012 < Formula`` for Flocker-0.1.2.
+     This should be changed.
 
-       .. code-block:: console
+     The version number is also included in the ``url`` part of the recipe and should be changed as appropriate.
 
-          sha1sum "dist/Flocker-${VERSION}.tar.gz"
-          ed03a154c2fdcd19eca471c0e22925cf0d3925fb  dist/Flocker-0.1.2.tar.gz
+     Update the ``sha1`` checksum. Retrieve it with ``sha1sum``:
 
-     - Commit the changes and push:
+     .. code-block:: console
+
+         sha1sum "dist/Flocker-${VERSION}.tar.gz"
+         ed03a154c2fdcd19eca471c0e22925cf0d3925fb  dist/Flocker-0.1.2.tar.gz
+
+     Download and run the `mkpydeps`_ script.
```

Comment:
Apart from the specific instruction `mkpydeps flocker`, everything is covered by the above step:

```
#. Create (if necessary) and activate the Flocker release virtual environment:

   .. note:: The following instructions use `virtualenvwrapper`_ but you can use `virtualenv`_ directly if you prefer.

   .. code-block:: console

      mkvirtualenv flocker-release-${VERSION}
      pip install --editable .[release]
```

Note that `setup.py` 's release section has added the new dependency.